PWGLF/Tasks/Strangeness/derivedlambdakzeroanalysis.cxx

```diff
@@ -318,7 +318,7 @@ struct derivedlambdakzeroanalysis {
     histos.get<TH1>(HIST("hEventSelection"))->GetXaxis()->SetBinLabel(12, "Above max occup.");
 
     histos.add("hEventCentrality", "hEventCentrality", kTH1F, {{100, 0.0f, +100.0f}});
-    histos.add("hCentralityVsNch", "hCentralityVsNch", kTH2F, {axisCentrality, {500, 0.0f, +2000.0f}});
+    histos.add("hCentralityVsNch", "hCentralityVsNch", kTH2F, {axisCentrality, {500, 0.0f, +5000.0f}});
```

Comment:
Hi @romainschotter , thanks! Would it perhaps not be advisable to make this axis configurable? :-D

Reply:
Done!